src/components/tools/drawing-tool/drawing-toolbar-buttons.tsx

```diff
@@ -104,11 +105,12 @@ export const StampModeButton: React.FC<IStampModeButtonProps> = ({
     <Tooltip title={title} {...tooltipOptions}>
       <div className={buttonClasses({ modalButton: "stamp", selected })} {...handlers}>
         <img src={stamp.url} draggable="false" />
-        <div className="expand-collapse" onClick={handleExpandCollapseClick}>
-          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 7 7" width="7" height="7">
-            <polygon points="0,7 7,7 7,0"/>
-          </svg>
-        </div>
+        {stampCount > 1 &&
+          <div className="expand-collapse" onClick={handleExpandCollapseClick}>
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 7 7" width="7" height="7">
+              <polygon points="0,7 7,7 7,0"/>
+            </svg>
```

Comment:
Any reason not too store this in an SVG file that gets imported in case it's needed elsewhere?  Granted, it's so darn simple that that approach might be overkill.

Reply:
I see there are a few of these cases where small SVGs are defined inline.  This seems like the one that might be reusable.